pipeline/__init__.py

```diff
@@ -3,7 +3,7 @@
 from model.single_doc import LexRankModel
 
 from dataset.st_dataset import SummDataset
-from dataset.non_huggingface_datasets import ScisummnetDataset
+from dataset import ScisummnetDataset
```

Comment:
Why does this depend on a specific dataset here?

Reply:
`ScisummnetDataset` data instances are formatted a little strangely, it's a single-doc dataset but each instance source is a list containing two items: the document xml and citations in json format or something like that. See line 357 in `dataset/dataset_loaders.py` (the `_process_data` function for `Scisummnet`). Not sure if this was intentional, I just did a bandaid patch on it for now assuming we might fix it later

Reply:
Okay, let's fix that in another thread. Merging this one now